mirrord-operator/templates/cluster-role.yaml

```diff
@@ -25,9 +25,11 @@ rules:
   - list
   - watch
 - apiGroups:
+  - ""
   - batch
   resources:
   - jobs
```

Comment:
can remove jobs then, no?
also, don't forget to add it to mirrord setup

Reply:
Not yet, copy-target is still in the old way, another pr and then we can.